.travis.yml

```diff
@@ -2,8 +2,9 @@ language: python
 python:
   - "2.7"
 env:
-  - TOX_ENV=py27-django18
   - TOX_ENV=py27-django111
+  - TOX_ENV=py27-django220
```

Comment:
ah this is why it's running `py27-django220` this should probablt be `py37-django111` instead?